deel-2-flexbox-layout/opdracht-1/styles/styles.css

```diff
@@ -1 +1,37 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+.body-container {
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+}
+
+header {
+    flex-grow: 0;
+    flex-shrink: 0;
+    flex-basis: 150px;
+}
+
+footer {
+    flex-grow: 0;
+    flex-shrink: 0;
+    flex-basis: 150px;
+}
+
+.container {
+    display: flex;
+    flex-direction: row;
+    flex: 1;
+}
+
+.article {
+    flex: 3;
+}
+
+.nav {
+    flex-grow: 1;
+    flex-shrink: 0;
+}
+
+aside {
+    flex-grow: 1;
+    flex-shrink: 0;
+}
```

Comment:
Code niet helemaal hetzelfde. Overall hoef je geen classes te maken voor elementen die maar 1 keer voorkomen. Verder had je de property "order" kunnen gebruiken om de article en de aside van plaats te laten wisselen.

Reply:
'Order' is nieuw voor mij. Wel handig. Had ook viewport width moeten gebruiken. Wist ook niet van dat bestaan af :D